src/apps/containers/components/ContainerCard.tsx

```diff
@@ -47,53 +60,62 @@ const ContainerCard: React.FC<Props> = ({
   return (
     <Grid item xs={4}>
       <Card onClick={() => goTo(url)} className={classes.card}>
-        <CardContent>
-          <Typography
-            noWrap
-            variant="body1"
-            color="textSecondary"
-            gutterBottom
-            data-testid={`shortCode-${index}`}
-          >
-            {short_code}
-          </Typography>
-          <Typography
-            className={classes.containerName}
-            noWrap
-            variant="h5"
-            data-testid={`name-${index}`}
-          >
-            {name}
-          </Typography>
-          <Typography
-            noWrap
-            variant="body2"
-            color="textSecondary"
-            data-testid={`owner-${index}`}
-          >
-            {owner_type}/{owner}
-          </Typography>
-          <Typography
-            noWrap
-            variant="body1"
-            component="p"
-            data-testid={`description-${index}`}
-          >
-            {description}
-          </Typography>
-        </CardContent>
-        <CardActions>
-          <Button
-            to={{ pathname: url, state: { prevPath: location.pathname } }}
-            component={Link}
-            size="small"
-            variant="text"
-            color="primary"
-            data-testid={`viewButton-${index}`}
-          >
-            View
-          </Button>
-        </CardActions>
+        <div className={classes.details}>
+          <CardContent>
+            <Typography
+              noWrap
+              variant="body1"
+              color="textSecondary"
+              gutterBottom
+              data-testid={`shortCode-${index}`}
+            >
+              {short_code}
+            </Typography>
+            <Typography
+              className={classes.containerName}
+              noWrap
+              variant="h5"
+              data-testid={`name-${index}`}
+            >
+              {name}
+            </Typography>
+            <Typography
+              noWrap
+              variant="body2"
+              color="textSecondary"
+              data-testid={`owner-${index}`}
+            >
+              {owner_type}/{owner}
+            </Typography>
+            <Typography
+              noWrap
+              variant="body1"
+              component="p"
+              data-testid={`description-${index}`}
+            >
+              {description}
+            </Typography>
+          </CardContent>
+          <CardActions>
+            <Button
+              to={{ pathname: url, state: { prevPath: location.pathname } }}
+              component={Link}
+              size="small"
+              variant="text"
+              color="primary"
+              data-testid={`viewButton-${index}`}
+            >
+              View
+            </Button>
+          </CardActions>
+        </div>
+        <div>
+          {name === "CIEL" ? (
+            <VerifiedUser className={classes.trustedIcon} />
```

Comment:
Instead of defining this in two places with separate classes, this should be turned into a component in `src/components`, e.g., a `VerifiedSource` component.